app/forms/candidate_interface/equality_and_diversity/ethnic_background_form.rb

```diff
@@ -30,17 +30,36 @@ def save(application_form)
       other_background_present = ethnic_background == EthnicBackgroundHelper::OTHER_ETHNIC_BACKGROUNDS[group].first && other_background.present?
 
       background = other_background_present ? other_background : ethnic_background
+      ethnicity_type = hesa_ethnicity_type(ethnic_background)
```

Comment:
Doesn't look like we use `ethnicity_type` directly in this method, maybe just fetch `ethnicity_code = ...` and use that? V minor

Reply:
Woops - `ethnicity_type` and `hesa_ethnicity_type` should actually be renamed to `ethnicity_value` and `hesa_ethnicity_value` respectively. This variable is used in a couple of places (lines 39 & 44) - I'll  make the change